dataactbroker/handlers/fileHandler.py

```diff
@@ -163,14 +164,17 @@ def submit(self,name,CreateCredentials):
             if not fileNameMap and existingSubmission:
                 raise ResponseException("Must include at least one file for an existing submission",
                                         StatusCode.CLIENT_ERROR)
-
             if not existingSubmission:
-                # Existing submissions do not recreate D jobs
+                # Don't add external files to existing submission
                 for extFileType in FileHandler.EXTERNAL_FILE_TYPES:
                     if extFileType == "award":
                         filename = CONFIG_BROKER["d2_file_name"]
                     elif extFileType == "award_procurement":
                         filename = CONFIG_BROKER["d1_file_name"]
+                    elif extFileType == "awardee_attributes":
+                        filename = CONFIG_BROKER["awardee_file_name"]
+                    elif extFileType == "sub_award":
+                        filename = CONFIG_BROKER["sub_award_file_name"]
```

Comment:
Not a big deal, but now that there are four branches, would it make sense to move this into a mapping? Something like:
```python
FILE_TYPE_TO_NAME = {
    "award": CONFIG_BROKER["d2_file_name"],
    "award_procurement": CONFIG_BROKER["d1_file_name"],
    ...
}
...
filename = FILE_TYPE_TO_NAME[extFileType]
```

It'd also be a good idea to standardize these names (if possible) so that we could use, e.g.
```python
filename = CONFIG_BROKER[extFileType + "_file_name"]
```